src/main/java/cc/hyperium/integrations/betterchatfilter/BetterChatFilter.java

```diff
@@ -59,11 +59,7 @@ public BetterChatFilter() {
     } catch (Exception e) {
       e.printStackTrace();
       // After failing to download file, make arraylist to make up for it.  
-      ArrayList rawBadwords = new ArrayList<String>();
-      rawBadwords.add("fuck");
-      rawBadwords.add("shit"); 
-      rawBadwords.add("damn");
-      rawBadwords.add("bitch");
+      String rawBadwords = "fuck shit bitch damn";
```

Comment:
You don't need to make bad word list in string separated by space and split it later on